src/errors.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+export interface AzFuncError {
+    /**
+     * System errors can be tracked in our telemetry
+     * User errors cannot be tracked in our telemetry because they could have user information (users can still track it themselves in their app insights resource)
+     */
+    isAzureFunctionsSystemError: boolean;
```

Comment:
Now that this property is renamed, is this property used by the host at all or just in our codebase? I don't think it is and I couldn't find `isAzureFunctionsInternalException` searching in the host repo, but just making sure

Reply:
This is just used in our codebase, not the host. This is the main place it's used:

https://github.com/Azure/azure-functions-nodejs-worker/blob/v3.x/src/setupEventStream.ts#L97